### ISSUE TITLE
The FTP menu..

### ISSUE THREAD

**Issue by wdlkmpx** [masneyb/gftp#44]
I'm reimplementing the menus with GtkUIManager

FTP menu with gtkItemFactory (gtk1) deprecated in gtk2.4 but still working in 2.24
![m1](https://user-images.githubusercontent.com/10052849/73552367-11e75800-4483-11ea-8786-5216f3718f30.png)

Work in progress
![m2](https://user-images.githubusercontent.com/10052849/73552374-157adf00-4483-11ea-89a5-acd96f89f7b8.png)

The "Window 1" & "Window 2" menu items.. are confusing... there is only 1 window and 2 panes or something: "Local" and "Remote".. when you select Window 1 or Window 2, nothing really happens, you would think that are 2 windows and one is hidden.

I think those 2 items should be removed.. or not implemented in the new menu.

Ascii vs Binary... this requires a GtkRadioActionEntry and I still don't know how to use it.
It should probably be somewhere else (FTP -> Preferences (Options dialog) -> FTP -> Mode)

Hmm it's a multiprotocol application.. should the FTP menu be renamed to something else?

**Comment by wdlkmpx** [comment]
I admit I never read the USERS-GUIDE

There are 2 windows
- The local directory window
- The remote directory window

The menu actions don't do anything actually, I see that a single character is added or removed from a label: '*' The focus doesn't change.

I don't know perhaps some terms should change: "pane" or something. I see in pcmanfm an option for a "Dual Pane Mode".. I see 2 directory listings.. some possible changes ahead and I certainly can not update documentation and stuff..

**Comment by wdlkmpx** [comment]
GtkUIManager was deprecated in 3.10 in favor of GtkBuilder, and icons in menus were also deprecated, gtk apps in gtk4 will have very simple menus without icons.. but when I run QBittorrent I see a lot of icons in menus, so many icons...

![q](https://user-images.githubusercontent.com/10052849/73556288-1ebb7a00-448a-11ea-8fc5-fc5b2625724f.png)

**Comment by wdlkmpx** [comment]
After performing more tests and connecting to different servers while "switching" between windows...

I found that Window 1 and Window 2 may exist.. in another dimension. I found that the text in the comboboxes change.

I would seem that you actually have 2 'main' window... but only 1 ftp connection is allowed between the two.. kind of useless.

I say that this is too confusing, and probably the initial part of some unimplemented feature. I'm noticing many strange things... no wonder nobody has been able to update the GUI.
